package.json

```diff
@@ -126,6 +126,7 @@
     "typeface-oxygen-mono": "1.1.13"
   },
   "resolutions": {
+    "**/focus-trap": "6.2.1",
```

Comment:
Can we add a comment on why we need that? If we just keep adding resolutions without documented reasoning, they won't ever disappear..

Reply:
Good point. I'll add a note to the read-me.

Reply:
Done